utils/fuzz/fuzz_targets/priority_set.rs

```diff
@@ -144,6 +162,14 @@ fn fuzz(input: Vec<FuzzInput>) {
             FuzzInput::PartialCmp => {
                 set.iter().partial_cmp(PrioritySet::new().iter());
             }
+
+            FuzzInput::PutSamePriority { items, priority } => {
+                // Insert multiple items with same priority to trigger Entry::partial_cmp
+                for item in items.iter().take(3) {
```

Comment:
I think you can do `into_iter` here and remove the clone from expected_items.insert